scripts/backend_tests.py

```diff
@@ -211,7 +211,24 @@ def _get_all_test_targets(test_path=None, include_load_tests=True):
     """
     def _convert_to_test_target(path):
         """Remove the .py suffix and replace all slashes with periods."""
```

Comment:
Please update the docstring with the correct args, returns, etc. 

Please also change the names of the function / args / variables to reflect what this method is now doing.

Reply:
Done.

Reply:
Done.